harness/determined/searcher/_search_method.py

```diff
@@ -393,7 +393,7 @@ def progress(self, searcher_state: SearcherState) -> float:
         Returns:
             float: Experiment progress as a float between 0 and 1.
         """
-        pass
+        return 1.0
```

Comment:
why are you returning values from `@absractmethod`s?

Reply:
I could revert this. I wrote a version of an example using a Custom Searcher, but didn't commit it as it has its own issues that I don't recommend users mess with. I overwrote these because it annoyed me that I must overwrite all of these methods lest they fail when all I wanted was a Custom Searcher to iterate over a list of model versions. Probably a change for another time.